### PR TITLE
Extending IIIF Manifest Support for Resource Metadata

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Addresses #18 by:
- Exposing metadata in implementing the `MetadataBuilder` and locally overriding the `ManifestServiceLocator`
- Exposing the license restrictions for the resource
- Exposing links to other resources (using `see_also`)

### CODE REVIEW

File: app/services/manifest_builder/metadata_builder.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      def label
+        if respond_to?("#{@attribute}_label".to_sym)
+          send("#{@attribute}_label".to_sym)
+        else
+          @attribute.to_s.titleize
+        end
+      end
+
+      def date_value
+        # binding.pry
```
Comment (escowles) [inline app/services/manifest_builder/metadata_builder.rb]: Please remove this and the other commented-out `binding.pry` below

File: app/services/manifest_builder.rb [hunk 2]
```diff
@@ -1,49 +1,77 @@
...
     end
 
+    ##
+    # Helper method for the resource manifest
     def manifest_url
       helper.polymorphic_url([:manifest, resource])
     end
 
+    ##
+    # Retrieves the
```
Comment (escowles) [inline app/services/manifest_builder.rb]: Do you want to complete this comment?

**Comment by escowles** [comment]
This looks good to me — just a couple very minor things to cleanup.

**Comment by tpendragon** [comment]
Maybe open an issue to move this functionality to the iiif_manifest gem?

**Pull request merged**